CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - A default role `Network Graph Viewer` has been added that provides sufficient privileges to display network graphs.
 - A new command `roxctl central login` has been added that allows to use a user's token within roxctl instead of an API token or admin password.
 - ROX-15447: A new `DelegatedRegistryConfig` API at `/v1/delegatedregistryconfig` has been added that provides dynamic configuration for local registry scanning (replaces `ROX_FORCE_LOCAL_IMAGE_SCANNING`).
+- A new environment variable `ROX_DISABLE_SIGNATURE_FETCHING` has been added to Central and Sensor which stops fetching image signatures in case the signature verification feature shall not be used.
```

Comment:
nit: what would happen if values are not aligned in Central and Sensor?

Reply:
So, in case only Sensor has `ROX_DISABLE_SIGNATURE_FETCHING=true`, it'd lead to cluster-local images won't ever get signatures, since Central cannot fetch them.

In case Central has `ROX_DISABLE_SIGNATURE_FETCHING=true`, it'd lead to all non-cluster-local images to never get signatures, but the cluster local ones will still have them.